resources/views/landing.blade.php

```diff
@@ -0,0 +1,4 @@
+<!-- Main Content -->
+@extends('layout.welcome')
+@section('content')
```

Comment:
Please copy the content of `main-content.blade.php` here between `section` and `stop`.

Reply:
Then, you can remove 'main-content.blade.php'